bodhi/tests/messages/schemas/test_base.py

```diff
@@ -40,14 +38,11 @@ class JsonClass(object):
             def __json__(self):
                 return {'my': 'json'}
 
-        self.assertEqual(
-            {'my': 'json'},
-            base.FedMsgEncoder().default(JsonClass())
-        )
+        assert {'my': 'json'} == base.FedMsgEncoder().default(JsonClass())
 
     def test_default_other(self):
         """Fallback to the superclasses' default."""
-        self.assertRaises(
+        pytest.raises(
```

Comment:
This was broken before migrating to PyTest, let's fix it.

Both `unittest.TestCase.assertRaises()` and `pytest.raises()` are context managers, i.e. this piece should become:

````
        with pytest.raises(TypeError):
            base.FedMsgEncoder().default(object())
````

Reply:
Will change it, but can you explain a little further how this is broken? What benefit does this change give us?